src/templates/calendar/calendar.js

```diff
@@ -17,14 +17,19 @@ import './calendar.scss';
  *   The component.
  */
 function Calendar({ content }) {
-  dayjs.extend(localizedFormat);
-
   const [currentDate, setCurrentDate] = useState(null);
   const [translations, setTranslations] = useState();
 
   const { backgroundColor, hasDateAndTime, title, events } = content;
   const classes = `template-calendar ${backgroundColor}`;
 
+  /**
+   * dayjs with localizedformat.
+   */
+  useEffect(() => {
```

Comment:
Why is this not a part of the useEffect below, are they not related?

Reply:
I think you should just gather all the useEffects that are dependent on [] into one to make it easier to read.